src/ContainerInterface.php

```diff
@@ -17,7 +19,7 @@ interface ContainerInterface
      *
      * @return mixed Entry.
      */
-    public function get($id);
+    public function get(string $id);
```

Comment:
Should we require PHP 8 here so we can include a `mixed` return?  I'm tempted to say yes, although admittedly `mixed` is not the most descriptive type.

Reply:
We'll be able to get a stepped migration from users still on PHP 7 to users on PHP 8 if we omit it, which, as somebody looking at updating a library already, is going to be useful. Otherwise, libraries cannot even look at adopting it until PHP 8 is their minimum supported version.

Reply:
Maybe this qualifies for a follow-up 2.1 release, just bumping to 8.0 and adding that mixed return?